appveyor.yml

```diff
@@ -130,10 +213,13 @@ artifacts:
   name: IBM.WatsonDeveloperCloud.NaturalLanguageUnderstanding.v1
 - path: '\src\IBM.WatsonDeveloperCloud.NaturalLanguageClassifier.v1\bin\$(configuration)\*.nupkg'
   name: IBM.WatsonDeveloperCloud.NaturalLanguageClassifier.v1
+before_deploy:

```

Comment:
We have this set in the `deploy` step in other repos. Should this be `deploy_script`?

Reply:
AppVeyor is set up to automatically deploy to NuGet in the deploy step. This will run the semantic-release command before it runs the deploy to NuGet.